src/Model/Activable/Activable.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * This file is part of the KnpDoctrineBehaviors package.
+ *
+ * (c) KnpLabs <http://knplabs.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Knp\DoctrineBehaviors\Model\Activable;
+
+
+/**
+ * Activable trait.
+ *
+ * Should be used inside entity where you need to track which user created or updated it
```

Comment:
This PHPDoc is completely confusing me. If you need to track, which user created or updated it - use [blameable behavior](https://github.com/KnpLabs/DoctrineBehaviors#blameable). If you just copypasted it, I have bad news - it must be changed.